app/lib/efile/az/az140.rb

```diff
@@ -68,7 +68,7 @@ def calculate
       end
 
       def refund_or_owed_amount
-        calculate_line_79
+        calculate_line_79 - calculate_line_80
```

Comment:
line 79 is just the refund, so i subtracted the amount owed on line 80 from it, which i believe is the intent of this method. @rickreyhsig could you double check in case I misunderstood? if this is correct, I think we might need a similar change on the NY side cuz I think it also has refund and amount owed on separate lines iirc

Reply:
@loumoore the way I interpreted this initially, is that if `line63 - line78` would get written to line79 regardless if positive or negative, but coming back a second time it seems indeed to make more sense that only one would box filled would make more sense since there is either a refund or an amount owed.  Thanks for the change.  It seems the 201 form math may capture the intent of method which is returning (+) if refund and (-) if owed 

```
      def refund_or_owed_amount
        line_or_zero(:IT201_LINE_76) - line_or_zero(:IT201_LINE_62)
      end
```